Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -211,7 +212,12 @@ class BrowserViewController: UIViewController {
             footer.addSubview(toolbar!)
             toolbar?.tabToolbarDelegate = self
 
-            let theme = Theme.of(tabManager.selectedTab)
+            let theme: Theme
+            if let tab = tabManager.selectedTab {
+                theme = Theme.of(tab)
```

Comment:
This takes an optional so could do something like:

```
let fallbackTheme = PrivateBrowsingManager.shared.isPrivateBrowsing ? .private : .regular
let theme = Theme.of(tab) ?? fallbackTheme
toolbar?.applyTheme(theme)
```

I just have a hatred towards `if` statements, so up to you 😄

Reply:
Haha the problem is that `Theme.of(tab)` takes an optional and returns `.regular` if there's no tab.

I could also update that instead to use the PrivateBrowsingManager when the tab given is nil